packages/input-output-logger/README.md

```diff
@@ -45,6 +45,7 @@ npm install --save @middy/input-output-logger
 ## Options
 
 `logger` property accept a function (default `console.log`)
+`exclude` property accept an array of paths to exclude from the log (default `[]`)
```

Comment:
I initially had a feeling that this would exclude the entire log message if an "exclude" path would match it.

Going through the code it seems that this works a bit more as a "field redaction" mechanism (e.g. to remove passwords from logs).

If that's the case, maybe we can be a bit more explicit here with something like this:

>  `exclude` property accepts an array of paths that will be used to remove particular fields from the logged objects. This could serve as a simple way to redact sensitive data from logs (default `[]`).

I was also suggesting that changing `exclude` to something more explicit can help to clarify the scope here. What do you think?

Reply:
Done [here](https://github.com/middyjs/middy/pull/507/commits/b213c25704ce52b3958a9ca8c23a389394952bdc)

Reply:
Awesome, thanks!